cities/dresden.json

```diff
@@ -0,0 +1,160 @@
+{
+    "features": [
+        {
+            "geometry": {
+                "coordinates": [
+                    13.7564374,
+                    51.0706049
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "location": "Alaunplatz",
+                "opening_hours": "Th 09:00-17:00; Su 08:00-12.00",
+                "title": "Markt am Alaunplatz"
+            },
+            "type": "Feature"
+        },{
+            "geometry": {
+                "coordinates": [
+                  13.7403693,
+                  51.0614998
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "location": "13.7403693, 51.0614998",
+                "opening_hours": "Sa 09:00-13:00",
+                "title": "Dresdner Bauernmarkt Königstraße"
+            },
+            "type": "Feature"
+        },{
+            "geometry": {
+                "coordinates": [
+                  13.7596107,
+                  51.1092368
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "location": "Markt in Dresden Hellerau",
+                "opening_hours": "Fr 09-16:00",
```

Comment:
The time format is not correct here. Should be `Fr 09:00-16:00`.
Please update the commit using `git commit --fixup=d3ee6b` followed by `git rebase -i --autosquash master`. Then update this branch via `git push --force origin market-data-dresden`.